repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/ratelimiting/CheckRateLimitWConfig.groovy

```diff
@@ -54,33 +50,16 @@ class CheckRateLimitWConfig extends ReposeValveTest {
     def cleanup() {
         waitForLimitReset()
     }
-    /*
-    REP-2181 Fix JSON support for rate limits get limits call
-
-*/
 
+    //2233
```

Comment:
It used to have an ignore tag indicating it was for bug 2233. I probably had it there so I could find the test easily after closing intellij. 

On October 21, 2015 2:45:07 PM CDT, William Scheidegger <notifications@github.com> wrote:
>>  
>> +    //2233
>
>What is this comment for?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rackerlabs/repose/pull/1398/files#r42672185

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.